portfolio/src/main/webapp/index.html

```diff
@@ -94,7 +60,7 @@ <h1 class ="titles">Photography</h1>
       <h1 class="titles">Blog</h1>
       <div class="container">
         <div class="cards">
-          <a class="card" href="#">
+          <a class="card">
```

Comment:
What's the motivation behind putting anchor tags for all these when there's no link? 

This is discouraged via best practices for a11y (https://a11yproject.com/posts/creating-valid-and-accessible-links/ for example). 

Essentially an anchor tag without a link means it's probably not the right element for the job. 

I'm guessing you could just use a div with some extra styling to accomplish what you want here.

Reply:
I was going to link the cards to another page. I just left it there until I made the pages. I should have put a TODO or just left them out for clarity.

Reply:
Ah that makes more sense. Yeah typically a TODO or leaving the href there is preferable but since we're iterating quickly it's fine for now.